egs/csj/s5/local/chain/tuning/run_tdnn_7k.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+
+set -e
+
+# configs for 'chain'
+affix=
+stage=8
+train_stage=-10
+get_egs_stage=-10
+speed_perturb=false
+dir=exp/chain/tdnn_csj  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+decode_nj=50
+
+# training options
+num_epochs=4
+initial_effective_lrate=0.001
+final_effective_lrate=0.0001
+leftmost_questions_truncate=-1
+max_param_change=2.0
+final_layer_normalize_target=0.5
+num_jobs_initial=1
+num_jobs_final=1
+minibatch_size=128
+frames_per_eg=150
+remove_egs=false
```

Comment:
better to have remove_egs to default to true.  And for the frames_per_eg I recommend 150,140,100
for minibatch_size I recommend 128,64
(they are both comma-separated strings).
I think it would be best if you copied the current Swbd configuration, which is as follows:

```
 relu-batchnorm-layer name=tdnn1 dim=625                                                                                                       
  relu-batchnorm-layer name=tdnn2 input=Append(-1,0,1) dim=625                                                                                  
  relu-batchnorm-layer name=tdnn3 dim=625                                                                                                       
  relu-batchnorm-layer name=tdnn4 input=Append(-1,0,1) dim=625                                                                                  
  relu-batchnorm-layer name=tdnn5 dim=625                                                                                                       
  relu-batchnorm-layer name=tdnn6 input=Append(-3,0,3) dim=625                                                                                  
  relu-batchnorm-layer name=tdnn7 input=Append(-3,0,3) dim=625                                                                                  
  relu-batchnorm-layer name=tdnn8 input=Append(-3,0,3) dim=625                                                                                  
  relu-batchnorm-layer name=tdnn9 input=Append(-3,0,3) dim=625       
```